code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -425,7 +425,7 @@ GLOBAL_LIST_INIT(plastitanium_recipes, list(
 	name = "alien alloy"
 	icon = 'icons/obj/abductor.dmi'
 	icon_state = "sheet-abductor"
-	item_state = "sheet-abductor"
```

Comment:
This probably got removed in the deconflicting. You need to re-add this line.

Reply:
Apologies, it's fixed now. Thanks!  @SteelSlayer